plugins/woocommerce-blocks/assets/js/atomic/blocks/product-elements/product-details/style.scss

```diff
@@ -1,4 +1,4 @@
-.wp-block-woocommerce-product-details {
+.editor-styles-wrapper .wp-block-woocommerce-product-details {
```

Comment:
If this code is only necessary in the editor, it would be great if we can move it to an `editor.scss` file in the same directory and import it from `edit.tsx`, this way it's not loaded in the frontend.